app/Notifications/PPEForm.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notifiable;
+use Illuminate\Notifications\Notification;
+
+class PPEForm extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param \Illuminate\Notifications\Notifiable $notifiable
```

Comment:
This is a trait, not a class or interface, and so cannot be used for type hints. In Apiary we're just leaving off type hints for these; you might need to adjust the analyzers to shut up about it.